app/Auth/PhotoAuthorisationProvider.php

```diff
@@ -53,7 +53,7 @@ public function applyVisibilityFilter(FixedQueryBuilder $query): FixedQueryBuild
 			return $query;
 		}
 
-		$userId = Authorization::id();
+		$userId = Authorization::idOrNull();
```

Comment:
Shouldn't this be
```suggestion
		$userId = Authorization::idOrFail();
```
to resemble the old behavior?